Jenkinsfile

```diff
@@ -0,0 +1,189 @@
+/*
+Jenkins Pipeline is a suite of plugins which supports implementing and integrating continuous delivery pipelines into Jenkins.
+Pipeline provides an extensible set of tools for modeling delivery pipelines "as code" via the Pipeline DSL.
+More information can be found on the Jenkins Documentation page https://jenkins.io/doc/
+*/
+
+library 'github-utils-shared-library@master'
+@Library('github.com/connexta/cx-pipeline-library@master') _
+pipeline {
+    agent {
+        node {
+            label 'linux-small'
+            customWorkspace "/jenkins/workspace/${env.JOB_NAME}/${env.BUILD_NUMBER}"
+        }
+    }
+    environment {
+        ION_GPG_KEYRING = credentials('ion-releases-key')
+        DISABLE_DOWNLOAD_PROGRESS_OPTS = '-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
+        LINUX_MVN_RANDOM = '-Djava.security.egd=file:/dev/./urandom'
+        COVERAGE_EXCLUSIONS = '**/test/**/*,**/itests/**/*,**/*Test*,**/sdk/**/*,**/*.js,**/node_modules/**/*,**/jaxb/**/*,**/wsdl/**/*,**/nces/sws/**/*,**/*.adoc,**/*.txt,**/*.xml'
+        GITHUB_USERNAME = 'connexta'
+        GITHUB_REPONAME = 'ion-index-api'
+        GITHUB_KEY = 'ion-index-api-github-key'
```

Comment:
This file is exactly the same as https://github.com/connexta/ion-ingest-api, except for `GITHUB_REPONAME` and `GITHUB_KEY`.